_policies/08_editorial_board.md

```diff
@@ -7,11 +7,8 @@ excerpt: A draft of the policies of the editorial board.
 permalink: /editorial_board/
 ---
 
-These policies are currently a draft, and not yet in force.
```

Comment:
I kind of want to leave this disclaimer, until we have actual policies. We could just not link this file from anywhere now, so it's only directly accessible from the web page.

Reply:
But we're not doing anything which would require enforcing policies yet. I guess I'm fine with having it there if you want, but it seems like it just adds extra steps to the process without providing any benefit. Since we aren't open for submissions yet, who is there to complain if we don't follow policy or who is even monitoring it? It's not like these are particularly restrictive.

Reply:
I just wanted to have a place we can discuss them before we actually do anything with them.  Could put it somewhere else rather than the livecomsjournal.github.io.

Reply:
We aren't getting a ton of feedback from people. My perspective is that we should just post how we think we're going to be doing things, give everyone a chance to look at it/provide feedback, then consider it approved. I thought that's what this was, and it seemed like it would work well for me. But it's up to you -- if you want to do it some other way that's fine with me.